electron/config.ts

```diff
@@ -242,9 +242,17 @@ abstract class GlobalConfig {
     // Known places where Steam might be found.
     // Just add a new string here in case another path is found on another distro.
     const steamPaths = [
-      `${home}/.steam`,
-      `${home}/.var/app/com.valvesoftware.Steam/.local/share/Steam`,
-      '/usr/share/steam'
+      join(home, '.steam'),
```

Comment:
I believe this is overkill. Because we wont use this on Windows or Mac so the path will always use `slashes`.
`join` is nice to use when we have a path that is used across platforms.
Doing this everywhere gives no benefit and makes the code less legible imo.

Reply:
Yes, agreed. I'll remove it once I get home